src/Features/Core/Portable/SignatureHelp/AbstractSignatureHelpProvider.cs

```diff
@@ -105,6 +95,11 @@ private static (IList<SignatureHelpItem> items, int? selectedItem) Filter(IList<
             // adjust the selected item
             var selection = items[selectedItem.Value];
             selectedItem = filteredList.IndexOf(selection);
+            if (selectedItem == -1)
+            {
+                selectedItem = null;
+            }
```

Comment:
was this something you encountered?

Reply:
Probably did at some point, but seems this can now be removed. Thanks